assets/scss/base/form/_hint.scss

```diff
@@ -126,3 +127,8 @@ Styleguide Form Hint.list
     }
   }
 }
+
+.form-hint-list-item--indented {
+  margin-left: em(30);
+  text-indent: em(-30);
+}
```

Comment:
Nice, I like it :+1:  Front-ends not using the indicators won't be affected. But removing the vertical-align: middle doesn't work.